miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -474,6 +504,7 @@ internal object ErrorBridgeMessage {
     const val NO_IMPLEMENT_CUSTOM_PERMISSION =
         "The `MiniAppMessageBridge.requestCustomPermissions` $NO_IMPL"
     const val ERR_SHARE_CONTENT = "Cannot share content:"
+    const val ERR_UNIVERSAL_BRIDGE = "Cannot send the universal bridge:"
```

Comment:
```suggestion
    const val ERR_UNIVERSAL_BRIDGE = "Please check the String and try again."
```

Reply:
@khairul-alam-licon

Reply:
Yeah, we could use `"Cannot send json info:"` instead as a prefix like the other existing features. But, `"Please check the String and try again."` is also fine.
I'm giving screenshots of both cases when sending an error message from HostApp using the interface.

![Screen Shot 2022-12-01 at 16 51 43](https://user-images.githubusercontent.com/66930373/204996753-cac65d56-2f46-4c2a-93ab-8a636786caef.png)
![Screen Shot 2022-12-01 at 16 53 13](https://user-images.githubusercontent.com/66930373/204996759-5c961f81-a5d3-4b00-8aed-8e43e0d6c050.png)

In that case, you can ignore [this comment](https://github.com/rakutentech/android-miniapp/pull/524#discussion_r1036742817).